backend/docker_ops.py

```
@@ -640,3 +738,108 @@ async def validate_docker_on_target(self, target_host: str, ssh_user: str = "roo
+                volume_mount = self._parse_volume_definition(volume, compose_dir)
+                # Use guard clause to handle invalid volume mounts
+                if not volume_mount:
+                    continue
+                    
+                volumes.append(volume_mount)
+        
+        return volumes
+    
+    def _parse_volume_definition(self, volume: Any, compose_dir: str) -> Optional[VolumeMount]:
+        """Parse a single volume definition from compose data"""
+        import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)